templates/zerver/help/change-stream-accessibility.md

```diff
@@ -0,0 +1,95 @@
+# Change the accessibility of a stream
+
+If you are a Zulip realm administrator, you can change who can join your stream
+by changing the stream's accessibility.
+
+Please note that any users that were previously subscribed to the stream will
+remain subscribed to the stream after the stream's accessibility change.
+
+## Make a public stream private
+
+1. Click the cog (![cog](/static/images/help/cog.png)) next to **Streams** in
+the left sidebar. A tooltip labeled **Subscribe, add, or configure streams**
+will appear upon hovering above the cog.
+
+    ![Browse streams](/static/images/help/streams-1.png)
+
+2. The stream browser will now appear, showing all streams that you've subscribed
+to by default. If you wish to see streams that you aren't subscribed to, click
+the **All Streams** tab; the tab will turn gray upon doing so.
+
+    ![All streams](/static/images/help/all-streams.png)
+
+3. Locate the public stream that you want to make private in the stream browser.
+You can search for specific streams by entering the name of the stream in the
+**Filter streams** search box.
+
+    ![Filter streams box](/static/images/help/filter-stream.png)
+
+4. Click on the stream that you want to make private. The right side of the
+stream browser, labeled **Stream settings**, will now display that settings of
+the stream that you've selected.
+
+    ![Stream selected](/static/images/help/stream-selected.png)
+
+5. After making sure that your cursor is hovering over the **Streams settings**
```

Comment:
Well, the right half of the stream browser is labeled Stream Settings and not Streams settings. I thought you wanted to refer to that. Still, just a very minor change